halpybot/commands/edsm.py

```diff
@@ -299,32 +299,45 @@ async def cmd_coordslookup(ctx, args: List[str]):
     return await ctx.reply(f"{system} is {dist} LY from {xcoord}, {ycoord}, {zcoord}.")
 
 
-@Commands.command("diversion")
+@Commands.command("longdiversion", "longdiversions")
+async def cmd_longdiversion(ctx: Context, args: List[str]):
+    if args[0] == "--new":
+        args.insert(1, "--long")
+    else:
+        args.insert(0, "--long")
+    return await cmd_diversionlookup(ctx, args)
```

Comment:
This _feels_ hacky, I'd personally rather see that this calls the function from the module directly instead of it calling another command with the same context

Reply:
This is VERY hacky. refactor.